src/include/migraphx/compile_options.hpp

```diff
@@ -40,6 +41,8 @@ struct compile_options
 
     bool fast_math       = true;
     bool exhaustive_tune = false;
+    bool weight_streaming = false;
+    long streaming_budget = LONG_MAX;
```

Comment:
I would prefer this option to be called `max_memory`, so the user can specify a max memory they want the model to use.

Reply:
Changed, thanks